_pages/himyfairy.yaml

```diff
@@ -0,0 +1,5 @@
+---
+  githubHandle: himyfairy
+  pageUrl: https://himyfairy.github.io/
```

Comment:
This URL will only work if it does not include the front `https://` portion. Try replacing it with this `himyfairy.github.io/` and then merge! 🚢